developer_docs/README.md

```diff
@@ -1,7 +1,6 @@
 This folder contains documents intended for developers of the p5.js Web Editor. 
 
 ## List of Documents
-* [Getting Started](getting_started.md) - A place to get started!
```

Comment:
Can you make your pr feature wise, you are commiting two different changes in same pr.

Reply:
gotcha. While we're on this, the `getting_started.md` file doesn't really exist and the purpose of it is already served by this `README.md` file.